src/ast/value.rs

```diff
@@ -78,6 +78,7 @@ pub enum Value {
     /// OBJECT constant as used by Snowflake
     /// https://docs.snowflake.com/en/sql-reference/data-types-semistructured#object-constants
     ObjectConstant(Vec<ObjectConstantKeyValue>),
+    Array(Vec<Value>),
```

Comment:
added this `Value::Array(..)` to support values as `["x", "y"]` in sql options like `uris = ["https://docs.google.com/spreadsheets/d/1g3xwWi1r-Ln2VVwv4mswwmqyfMeoJglv-MS80ywASGI/edit#gid=0"]`